charts/delphai-deployment/templates/scaled-object.yaml

```diff
@@ -0,0 +1,28 @@
+{{ $enabled := or $.Values.autoscaling.rabbitmq.enabled }}
```

Comment:
The idea that we'll list all possible triggers here and then enable autoscaling when at least one is enabled